ldelf/dl.c

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2019, Linaro Limited
+ */
+#include <string.h>
+
+#include "dl.h"
+#include "ta_elf.h"
+
+#define RTLD_NOW        2
+#define RTLD_GLOBAL     0x100
```

Comment:
indent with tabs.
These should be defined in `ldelf/include/ldelf.h`

Reply:
OK